packages/core/src/components/slider/slider.tsx

```diff
@@ -44,6 +44,9 @@ export interface ISliderProps extends ISliderBaseProps {
 
     /** Callback invoked when the handle is released. */
     onRelease?(value: number): void;
+
+    /** Applied to `aria-label` prop of the slider Handle. */
+    handleAriaLabel?: string;
```

Comment:
According to the [MDN ARIA: slider role docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/slider_role), we should also support `aria-labelledby` in addition to `aria-label`... I'm thinking the API would look better if we forwarded a subset of handle props from Slider...

```suggestion
    /** Subset of handle props to apply to the child Handle. */
    handleProps?: Pick<HandleProps, "ariaLabel" | "ariaLabelledBy">;
```

Reply:
yep you're right, we should support `aria-labelledby`. Did it a little differently, see changes